scripts/documentation.js

```diff
@@ -73,21 +79,164 @@ const TREE = {
   ]
 }
 
-function buildChildDoc(inputPath, outputPath, config) {
+function _overrideHeading(nodes) {
+  const contents = ['Examples', 'Parameters'];
+  const mdContents = contents.map(text => {
+    return unified()
+      .use(markdown)
+      .parse(`__${text}__`);
+  });
+
+  return nodes.map(node => {
+    if (node.type === 'heading' &&
+      Array.isArray(node.children) &&
+      contents.includes(node.children[0].value)) {
+
+      const value = node.children[0].value;
+      const replacement = mdContents[contents.indexOf(value)].children[0];
+      return replacement
+    }
+
+    return node;
+  });
+}
+
+function _appendActionTypesAndUpdatersToActions(node, actionMap) {
+  // __Updaters__: [`visStateUpdaters.loadFilesUpdater`](../reducers/ui-state.md#uistateupdaterssetexportfilteredupdater)
```

Comment:
do you need this one or you can delete it?

Reply:
Need it, as an example of how the md should be composed